backend/store.py

```
@@ -252,18 +257,56 @@ def remove_from_unclustered(self, feedback_id: UUID, project_id: UUID):
         Behavior:
             Performs a no-op if the project does not exist or the feedback ID is not present in the set.
         """
-        if project_id in self.unclustered_feedback_ids:
-            self.unclustered_feedback_ids[project_id].discard(feedback_id)
+        key = str(project_id)
+        if key in self.unclustered_feedback_ids:
+            self.unclustered_feedback_ids[key].discard(feedback_id)
 
-    def clear_feedback_items(self):
+    def update_feedback_item(self, project_id: str, item_id: UUID, **updates) -> FeedbackItem:
```

Ruff (0.14.8)

Unused method argument: `project_id`

(ARG002)